InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -510,6 +514,21 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 
 		scene->scheduleUpdate();
 	}
+	else if (event->modifiers() == Qt::ControlModifier && event->key() == Qt::Key_F12)
+	{
+		event->accept();
+		if (ViewSwitcherMenu::isVisible())
+			ViewSwitcherMenu::hide();
+		else
+		{
+			QVector<Model::Node*> nodes;
+			for (auto view : target->scene()->viewItems())
+				nodes.append(new Interaction::ViewSwitcherEntry(view->name()));
+			for (int i = nodes.size(); i < 9; i++)
+				nodes.append(new Interaction::ViewSwitcherEntry("Empty slot " + QString::number(i)));
+			ViewSwitcherMenu::show(nodes, target);
```

Comment:
I hope the ViewSwitcherMenu takes ownership of the nodes here, and deletes them once it's deleted, otherwise we'll leak memory.